scripts/setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup
+
+setup(
+    name='securedrop',
+    version='1.0',
+    packages=['securedrop', 'securedrop.tests'],
+    url='https://github.com/aokellermann/securedrop.git',
+    license='MIT License',
+    author='John Nay, Antony Kellermann, Robert Liani',
+    author_email='John_Nay@student.uml.edu',
+    description='Secure file sharing command line application.',
+    python_requires='>=3.8',
+)
```

Comment:
You need to put in an `install_requires` section.

Reply:
Added the packages on my machine (for example 'pycryptodome>=3.9.9') in the intstall_requires section.

Reply:
I think this includes non-dependencies. You can generate dependency list with `pip freeze`.

Reply:
Changed. Thanks for the pip freeze suggestion!